gk/main.c

```diff
@@ -31,6 +33,8 @@
 
 #define GATEKEEPER_MAX_PKT_BURST (32)
 
+static struct gk_config *gk_conf;
```

Comment:
Why do you need a global variable here? If `gk_proc()` needs it, why does `gk_proc()` receive it as a parameter?Please give a sound reason. Global variables should be used sparely.